source/contracts/reporting/Market.sol

```diff
@@ -144,7 +144,8 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
     }
 
     function contribute(uint256[] _payoutNumerators, bool _invalid, uint256 _amount) public returns (bool) {
-        require(feeWindow.isActive());
+        require(getInitialReporter().getReportTimestamp() != 0);
+        require(!feeWindow.isOver());
```

Comment:
The change here is from:
```isActive()```
to
```!isOver()```

The new code is more permissive, and allows this code to proceed when FeeWindow has not started yet. This would allow crowdsourcers to be contributed to before the corresponding FeeWindow starts, such as immediately following an InitialReport?

Reply:
That's exactly correct. The idea being that any number of dispute rounds can occur in any time period but for the market to resolve it must last an entire fee window with no completed dispute.